src/css/components/_alert.scss

```diff
@@ -131,6 +131,10 @@
     @include justify-content(center);
 }
 
+.btn-type-BOTH .soft-button-image {
+    max-width: 50%;
+}
```

Comment:
Unfortunately I'm seeing similarly odd things in other cases with BOTH softbuttons. Notably it seems to occur with templated dynamic images specifically